sdk/resourcemanager/azure-resourcemanager-samples/src/main/resources/create_resources_with_msi.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
-/usr/bin/yes | sudo apt-get update
```

Comment:
why remove these commands？

Reply:
the curl line now install the azure-cli, replacing these